src/parser/parse_tree_cl.c

```diff
@@ -2216,52 +2214,55 @@ parser_new_node (PARSER_CONTEXT * parser, PT_NODE_TYPE node_type)
  *   node(in/out):
  */
 PT_NODE *
-parser_init_node (PT_NODE * node)
+parser_init_node (PT_NODE * node, PT_NODE_TYPE node_type, bool bFirst, bool bKeepLnCn)
```

Comment:
```suggestion
parser_init_node (PT_NODE * node, PT_NODE_TYPE node_type, bool is_first, bool keep_line)
```
Most of CUBRID codes do not follow MicroSoft code style, as it's simply is_first.

Reply:
Yes, thank you.
I will separate the function and rewrite it.